### PR TITLE
Updateio

### PR DESCRIPTION
**Pull request by m-matthies**

### CODE REVIEW

**Comment by Akodiat** [comment]
In general, this is a great feature, well done! The timeline will be very useful!

- [x] However, it seems some trajectories don't load correctly, the positions get messed up (sometimes the structure disappears entirely) and the `t=` becomes 0, -0 or NaN. Loading the same trajectories in the master branch works without issue. I'll send you an example.

Some minor things:

- [x] It would still be nice with a pause/play button
- [x] I think we can remove the indexing progress bar when it has reached 100%, maybe replace it with a text saying indexing done and maybe a index file download button?

File: index.html [hunk 1]
```diff
@@ -207,14 +207,48 @@
...
                 </button>
-                <button class="ribbon-button" onclick="view.toggleWindow('videoCreationWindow')">
-                    <span class="mif-video-camera mif-3x"></span>
-                    <span class="caption">Create video</span>
-                </button>
                 <button class="ribbon-button" onclick="view.saveCanvasImage()" title="Save current view as an image ('p')">
                     <span class="mif-photo-camera mif-3x"></span>
                     <span class="caption">Save image</span>
                 </button>
+                <button id="videoCreateButton" class="ribbon-button" onclick="view.toggleWindow('videoCreationWindow')" disabled="true">
```
Comment (Akodiat) [inline index.html]: You still want to be able to create "lemniscate" videos if you don't have a trajectory

**Review by Akodiat — CHANGES_REQUESTED** [review]
See earlier comments too :)

File: index.html (hunk 1 above)
Comment (m-matthies) [reply index.html]: seems this function is not working on main as well ?

**Pull request merged**